src/components/ExportCSV/ExportCSVRow/ExportCSVRow.module.css

```diff
@@ -8,4 +8,11 @@
   color: red !important;
   width: 10px;
   height: 10px;
-}
\ No newline at end of file
+}
+
+.green-check-icon,
+.red-cross-icon {
+  position: absolute;
+  left: 80px;
+  top: 40%;
+}
```

Comment:
Is this styling being applied? It doesn't look centered on my screen.
![image](https://user-images.githubusercontent.com/25851212/163474308-9eb881ee-126b-4a32-abc4-3869d7580c47.png)